utils/src/main/java/org/nervos/ckb/crypto/Hash.java

```diff
@@ -63,7 +65,7 @@ public static String blake2bString(String utf8String) {
     private static Blake2bDigest blake2bDigest;
     public static Blake2bDigest update(byte[] input) {
```

Comment:
Isn't this `Hash` class safe? it's used as a static helpers which holds a single static `blake2bDigest` instance. If we call `Hash.update(bytes)` from two different hashing places these the digest would be polluted - mixed with two different inputs.

Reply:
I agree, I don't think this is type safe, we either need synchronized keyword or remove the static instance.

That brings to this question: why are we keeping the static instance here?

Reply:
Perhaps just change to normal class and use instance.

Reply:
OK. I will refactor later.